src/Command/BackgroundCommand.php

```diff
@@ -24,11 +24,12 @@ class BackgroundCommand extends Command
      */
     private $backgroundExecute;
 
-    public function __construct(string $name = null)
+    public function __construct(?string $name = null)
     {
         parent::__construct($name);
 
-        parent::setCode([$this, 'background']);
+        parent::setCode(fn($input, $output) => $this->background($input, $output));
```

Comment:
I'm not sure what this is trying to fix. Agreed it is a more modern style, but the method signature for `setCode()` in symfony/console v7.3 is `setCode(callable $code)`, so the existing style is still supported. I couldn't find an obvious deprecation notice for it.

If this does need to change, then we would also need to change `\Phlib\ConsoleProcess\Command\BackgroundCommand::setCode(callable $code)`